[2022-06-21 14:22:58,751] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_22 manual__2022-06-21T14:22:49.367507+00:00 [queued]>
[2022-06-21 14:22:58,895] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_22 manual__2022-06-21T14:22:49.367507+00:00 [queued]>
[2022-06-21 14:22:58,895] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-06-21 14:22:58,895] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-06-21 14:22:58,895] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-06-21 14:22:58,949] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): START_COUNT_BY_DAY_22> on 2022-06-21 14:22:49.367507+00:00
[2022-06-21 14:22:58,972] {standard_task_runner.py:52} INFO - Started process 1901 to run task
[2022-06-21 14:22:59,017] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'DM-COUNT-account-turnover', 'START_COUNT_BY_DAY_22', 'manual__2022-06-21T14:22:49.367507+00:00', '--job-id', '618', '--raw', '--subdir', 'DAGS_FOLDER/dm_fill_account_turnover.py', '--cfg-path', '/tmp/tmpr9vbs1rf', '--error-file', '/tmp/tmpw5pc99mi']
[2022-06-21 14:22:59,035] {standard_task_runner.py:77} INFO - Job 618: Subtask START_COUNT_BY_DAY_22
[2022-06-21 14:22:59,221] {logging_mixin.py:109} INFO - Running <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_22 manual__2022-06-21T14:22:49.367507+00:00 [running]> on host c087b85f9f15
[2022-06-21 14:22:59,448] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=DM-COUNT-account-turnover
AIRFLOW_CTX_TASK_ID=START_COUNT_BY_DAY_22
AIRFLOW_CTX_EXECUTION_DATE=2022-06-21T14:22:49.367507+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-21T14:22:49.367507+00:00
[2022-06-21 14:22:59,474] {base.py:79} INFO - Using connection to: id: postgres. Host: postgres, Port: 5432, Schema: postgres, Login: ***, Password: ***, extra: {}
[2022-06-21 14:22:59,487] {dbapi.py:225} INFO - Running statement: CALL ds.fill_account_turnover_f (%(date)s::date);, parameters: {'date': '2018-01-22'}
[2022-06-21 14:23:00,631] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=DM-COUNT-account-turnover, task_id=START_COUNT_BY_DAY_22, execution_date=20220621T142249, start_date=20220621T142258, end_date=20220621T142300
[2022-06-21 14:23:00,736] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-21 14:23:01,382] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
